platform.md

```diff
@@ -735,43 +735,38 @@ Usage:
 #### `launcher`
 Usage:
 ```
-/cnb/process/<process-type> [<arg>...]
+/cnb/process/<process-type> [<args>...]
 # OR
-/cnb/lifecycle/launcher [--] [<cmd> <arg>...]
+/cnb/lifecycle/launcher [<cmd> <args>...]
 ```
 ##### Inputs
 | Input                              | Environment Variable | Default Value | Description                                               |
 |------------------------------------|----------------------|---------------|-----------------------------------------------------------|
 | `<app>`                            | `CNB_APP_DIR`        | `/workspace`  | Path to application directory                             |
 | `<layers>`                         | `CNB_LAYERS_DIR`     | `/layers`     | Path to layer directory                                   |
 | `<process-type>`                   |                      |               | `type` of process to launch                               |
-| `<direct>`                         |                      |               | Process execution strategy                                |
 | `<cmd>`                            |                      |               | Command to execute                                        |
 | `<args>`                           |                      |               | Arguments to command                                      |
 | `<layers>/config/metadata.toml`    |                      |               | Build metadata (see [`metadata.toml`](#metadatatoml-toml) |
 | `<layers>/<buildpack-id>/<layer>/` |                      |               | Launch Layers                                             |
 
-A command (`<cmd>`), arguments to that command (`<args>`), a working directory (`<working-dir>`), and an execution strategy (`<direct>`) comprise a process definition. Processes MAY be buildpack-defined or user-defined.
+A command (`<cmd>`), arguments to that command (`<args>`), and a working directory (`<working-dir>`) comprise a process definition. Processes MAY be buildpack-defined or user-defined.
```

Comment:
Do we need to make mention of the `$(env)` interpolation that the launcher provides? The RFC [mentions](https://github.com/buildpacks/rfcs/blob/main/text/0093-remove-shell-processes.md#using-environment-variables-in-a-process)
> If a buildpack or user includes `$(<env>)` in the command or args and `<env>` is the name of an environment variable set in the launch environment, the launcher will replace this string with the value of the environment variable after apply buildpack-provided env modifications and before launching the process.

Reply:
The unresolved questions [section](https://github.com/buildpacks/rfcs/blob/main/text/0093-remove-shell-processes.md#unresolved-questions) of the RFC asks if we need a mechanism for turning `$(env)` interpolation off... I guess we should resolve that in this PR? 

It would be awesome if others could chime in with real-life examples of how this might be used - the example given `CNB_INTERPOLATE=false` doesn't solve for cases where users might want to skip interpolating __some__ variables but not all. Should that be a requirement? IDK what is the answer here.

Reply:
Added a description of environment variable interpolation, and went all-in on the [Kubernetes convention](https://kubernetes.io/docs/tasks/inject-data-application/define-interdependent-environment-variables/) (replace if defined, don't replace if undefined, and use `$$(<env>)` to escape). It makes sense to me to follow the k8s convention. Happy to hear any feedback.